test/extensions/filters/http/grpc_web/grpc_web_filter_integration_test.cc

```diff
@@ -141,7 +164,13 @@ TEST_P(GrpcWebFilterIntegrationTest, UpstreamDisconnect) {
 
   ASSERT_TRUE(fake_upstream_connection_->close());
   response->waitForEndStream();
+  EXPECT_TRUE(response->complete());
+
+  // TODO(dio): Need to check each response body, should reflect the requirement mandated by the
+  // request "Accept" header.
   EXPECT_EQ("503", response->headers().getStatusValue());
+  EXPECT_NE(nullptr, response->headers().GrpcMessage());
```

Comment:
why there is not GrpcMessage header?  LocalReply text body should be converted to it.

Reply:
Sorry, this is "EXPECT_NE", hence we expect to have it in "grpc-message". I added the content expectation instead to make it clear.